src/index.js

```diff
@@ -45,10 +45,10 @@
 const helpers = require('./helpers');
 
 // Import the clients for each version supported by this package.
-const gapic = Object.freeze({
+const gapic = {
```

Comment:
Is it necessary to make this object `mutable` ?

Reply:
with freeze intellisense is not working.